scene/2d/multimesh_instance_2d.cpp

```diff
@@ -42,8 +42,24 @@
 Callable MultiMeshInstance2D::_navmesh_source_geometry_parsing_callback;
 RID MultiMeshInstance2D::_navmesh_source_geometry_parser;
 
+void MultiMeshInstance2D::_refresh_interpolated() {
+	if (is_inside_tree() && multimesh.is_valid()) {
+		bool interpolated = is_physics_interpolated_and_enabled();
+		multimesh->set_physics_interpolated(interpolated);
+	}
+}
+
+void MultiMeshInstance2D::_physics_interpolated_changed() {
+	VisualInstance::_physics_interpolated_changed();
```

Comment:
```suggestion
	Node2D::_physics_interpolated_changed();
```

Reply:
Oh yeah good point let me look.

Reply:
In this case the parent class is `Node2D` but it should call down to `Node` and it's protected so can be called.

It's just a stub so should be compiled out, but works if we add in later functionality. I likely forgot to call it on some of the other 2D nodes, but sensible to put in if it compiles.